Src/StdLib/MakeModuleList.py

```diff
@@ -17,7 +17,7 @@
 # Usage: ipy getModuleList.py <output directory>
 
 #--Imports---------------------------------------------------------------------
-import sys, nt, os
+import sys, os
```

Comment:
Do we still need this file? Couldn't we just use a static pyproj file with a glob pattern? Maybe something to look into for a separate PR...

Reply:
Yes, we actually do. This file checks for modules that don't work with IronPython and does not include them in the release area.